plugins/kubernetes-backend/package.json

```diff
@@ -39,6 +39,7 @@
     "@backstage/catalog-client": "workspace:^",
     "@backstage/catalog-model": "workspace:^",
     "@backstage/config": "workspace:^",
+    "@backstage/core-plugin-api": "workspace:^",
```

Comment:
Don't think this dependency is needed. It's frontend only :)

Reply:
I agree, not sure how this snuck in here but removed. Thanks.